src/proxy.rs

```diff
@@ -0,0 +1,22 @@
+//! Proxy services of validator
+//!
+
+use crate::deploy::Host;
+use anyhow::{Context, Result};
+use std::process::Command;
+
+/// Proxy RPC service
+pub fn rpc(host: &Host, local_port: u16) -> Result<()> {
+    let address = host.ipv4_address;
+    let user = &host.install_ssh_user;
```

Comment:
This allows user to click a link:

```suggestion
    let user = &host.install_ssh_user;
    println!("open up proxy at: http://localhost:{local_port:}");
```